src/content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -72,6 +104,14 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 			var fullStringPath = fileTextSpan[0].getAttribute("title");
 			this._filePath = new FilePath( fullStringPath );
 
+			// Get DOM name of this file for navigation using location hash
+			var previousSibling = this._domElement.previousElementSibling;
```

Comment:
Just please leave comment why are we fetching previous sibling in order to get hash for current element? I guess it is because you want to leave small margin before 'desired' element. If that is the case, are we handling the case when there is no previous sibling?